examples/demo/xtask/src/main.rs

```diff
@@ -13,6 +15,7 @@ struct Opt {
 #[derive(StructOpt)]
 enum Command {
     Build(xtask_wasm::Build),
+    OptimizedBuild(xtask_wasm::Build),
```

Comment:
Actually this is the opportunity to show the extensibility of the command line.

Make a new `Build` struct and flatten the xtask_wasm::Build inside. Add a field `--optimize` or something (you can allow the user to fine tune for example).

This can be done in a separate PR